org.filezillaproject.Filezilla.json

```diff
@@ -0,0 +1,78 @@
+{
+    "id": "org.filezillaproject.Filezilla",
+    "base-version": "master",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.24",
+    "sdk": "org.gnome.Sdk",
+    "command": "filezilla",
+    "rename-icon": "filezilla",
+    "copy-icon": true,
+    "rename-desktop-file": "filezilla.desktop",
+    "rename-appdata-file": "filezilla.appdata.xml",
+    "finish-args": [
+            "--share=ipc", "--socket=x11",
+            "--device=dri",
+            "--socket=wayland",
+            "--socket=session-bus",
+            "--filesystem=home",
+            "--share=network"
+    ],
+    "modules": [
+        "shared-modules/glu/glu-9.0.0.json",
+        {
+        "name": "wxWidgets",
+        "config-opts": [
+            "--with-opengl", 
+            "--with-libjpeg", 
+            "--enable-monolithic",
+            "--with-libtiff",
+            "--with-libpng",
+            "--with-zlib",
+            "--disable-sdltest",
+            "--enable-unicode",
+            "--enable-display",
+            "--enable-propgrid",
+            "--disable-webkit",
+            "--disable-webview",
+            "--disable-webviewwebkit",
+            "--with-expat=builtin",
+            "--with-libiconv=/usr" ],
+        "make-args": ["-j4"],
```

Comment:
Huh, good point - I just totally cargo-culted the whole section out [of CubicSDR](https://github.com/cjcliffe/CubicSDR-flatpak/blob/master/com.cubicsdr.App.json) because I'd had such annoyances with wxwidgets in the past. I'll drop that.